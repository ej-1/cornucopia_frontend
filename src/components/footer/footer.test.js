// Link.react.test.js
import React from "react";
import Footer from "../footer/footer";
import renderer from "react-test-renderer";

test("Footer gets rendered", () => {
  const component = renderer.create(<Footer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
