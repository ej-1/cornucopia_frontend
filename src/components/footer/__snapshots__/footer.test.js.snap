// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message gets rendered 1`] = `
<div
  className="footer"
>
  <nav
    className="navbar navbar-default navbar-fixed-bottom"
  >
    <div
      className="container"
    >
      <ul
        className="nav navbar-nav"
        role={null}
      >
        <p
          className="navbar-text"
          onSelect={null}
        >
          Copyright © 2018 CornucopiaTrader
        </p>
      </ul>
    </div>
  </nav>
</div>
`;

exports[`Footer gets rendered 1`] = `
<div
  className="footer"
>
  <nav
    className="navbar navbar-default navbar-fixed-bottom"
  >
    <div
      className="container"
    >
      <ul
        className="nav navbar-nav"
        role={null}
      >
        <p
          className="navbar-text"
          onSelect={null}
        >
          Copyright © 2018 CornucopiaTrader
        </p>
      </ul>
    </div>
  </nav>
</div>
`;
